### DISCUSSION TITLE
Dropdown component for multiple buttons / links

### DISCUSSION THREAD

**Discussion by Grayda** [Budibase/budibase#4936] [Ideas]
At work we use Freshservice, and next to their Send button they have a button that brings down a drop-up menu for performing additional actions on a reply or note.

![image](https://user-images.githubusercontent.com/4891563/158769696-ce181997-16fe-4c05-8fd6-d3d6f7371050.png)

And on my own in-house app I wrote (built with Laravel Nova), I have a similar option which lets me run actions from the table, without having a button for everything:

![image](https://user-images.githubusercontent.com/4891563/158770095-0dcdb52d-4135-4f42-812e-d830136908a4.png)

It'd be nice to have a component (like a Field Group or a Section or something) that you could nest buttons into and get something like this, as I've got a few "shortcut" buttons I'd like to have shown in a table without cluttering up the UI

**Comment by joebudi** [comment]
This is a great idea and pretty common in web dev.
